libs/designsystem/src/lib/directives/fit-heading/fit-heading.directive.ts

```diff
@@ -73,9 +73,14 @@ export class FitHeadingDirective implements OnInit, OnDestroy {
   }
 
   private observeResize(): void {
-    this.resizeObserverService.observe(this.elementRef, () => {
-      this.scaleHeader();
-    });
+    // Supress errors thrown from ResizeObserver as they do not have any impact
+    // on the users.
+    // @see {@link https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded}
+    try {
+      this.resizeObserverService.observe(this.elementRef, () => {
+        this.scaleHeader();
+      });
+    } catch (e) {}
```

Comment:
Nice! 

I think we should check the message field of the error as to not unintentionally mute other errors. 

The standard error message is "_ResizeObserver loop completed with undelivered notifications._" according to [the spec](https://www.w3.org/TR/resize-observer/#deliver-resize-error). According to [this guy](https://github.com/maslianok/react-resize-detector/issues/45#issuecomment-858810843) Chromium based browser might have another message set tho for some reason.

Which i believe is the one you might be seeing when testing from an Android based phone. 

So we might have to check both. 

So something like the below i guess 🤷 

```typescript 
catch(e) {
   if message is either chrome message or standard message: 
      do nothing 
   else: 
      throw(e); 
}

Reply:
Also if this turns out to give problems or not work - let's revert to the requestAnimationFrame solution. But i think this one might be the least intrusive solution for now.

Reply:
Reverted to the requestAnimationFrame due to time constraints. 
The new proposed solution did not catch the error, and we need to investigate the error object a bit more to find out how to filter it. Unfortunately the process is slow, as we have to build kirby, install it in a project, and run the emulator to get feedback, so we will have to investigate more at a later time.